acasclient/acasclient.py

```diff
@@ -1383,3 +1385,98 @@ def get_blob_data_by_value_id(self, valueId):
                                 .format(self.url, valueId))
         resp.raise_for_status()
         return resp.content
+    
+    def get_authors(self):
+        """
+        Get all authors
+
+        Returns:
+            a list of dict objects representing the authors
+        """
+        resp = self.session.get("{}/api/authors".format(self.url))
+        resp.raise_for_status()
+        return resp.json()
+    
+    def get_author_by_username(self, username):
+        """
+        Get author by username
+            
+        Args:
+            username (str): The username of the author to fetch
+
+        Returns:
+            a dict object representing the author
+        """
+        resp = self.session.get("{}/api/authorByUsername/{}".format(self.url, username))
+        resp.raise_for_status()
+        return resp.json()
+    
+    def create_author(self, author):
+        """
+        Create an author
+
+        Args:
+            author (dict): A dict object representing the author to create
+
+        Returns:
+            a dict object representing the new author
+        """
+        def hash_password(password):
```

Comment:
Just to be a little more specific about @iamanandkr request.  I think if we do create a utils files we should make them named after what they have in them.  I find a general utils.py name does not make the utils functions discoverable which leads to people re-inventing functionality without knowing that similar functions already exist.

Reply:
I'm going to pass on the utils file idea simply because this utility function isn't used anywhere else. I purposely defined it as an in-line function since it is only used in this one function.

I'll apply the type hints suggestion though. I really like those too.

Reply:
Decided to do the type hints in a separate PR since it will be a lot of changes. I'll do it along with f-strings.